Resources/Prototypes/_Sunrise/Entities/Effects/overlays.yml

```
@@ -0,0 +1,10 @@
+- type: entity
+  id: EffectNightVision
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)